plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -205,6 +207,9 @@
     private final JsonCodec<CommitTaskData> commitTaskCodec;
     private final TrinoCatalog catalog;
     private final HdfsEnvironment hdfsEnvironment;
+    // for Iceberg table orc bloom filter
+    public static final String ORC_BLOOM_FILTER_COLUMNS_KEY = "orc.bloom.filter.columns";
```

Comment:
Move next to the other static final fields

Reply:
Also do you plan on adding support to other engines, Spark, Flink, etc? It'd be nice to have these constants defined centrally in `org.apache.iceberg.TableProperties` if it'll get used elsewhere

Reply:
I'm not familiar with spark and Flink and have no plan support.